lib/student.rb

```diff
@@ -0,0 +1,12 @@
+require './person'
+
+class Student < Person
```

Comment:
**cannot load such file -- ./person (LoadError)**

![image](https://github.com/Grandi0z/school_library/assets/78251927/f3611774-640a-4b8c-9f9e-931c976125c2)
```
/lib
    | person.rb
    | student.rb
...
```
As mentioned by the previous reviewer, please ensure that our app works smoothly without errors. 🤗🛠, 

To get rid of the errors, I noticed that the files `person.rb` and `student.rb` are both in the same directory, I would suggest that we use `require_relative` which is used to load files in the same dir. 

i.e.
```ruby 
require_relative 'person'
```
`require` is used to load external files.  🤓🛠

[Understanding Require vs. Require_relative vs. Require_all](https://medium.com/@ellishim/understanding-require-vs-require-relative-vs-require-all-80e3b26d89e6)

Reply:
Thank for the tips